pkg/services/oracle/neofs/neofs.go

```diff
@@ -47,7 +47,7 @@ var (
 // Get returns a neofs object from the provided url.
 // URI scheme is "neofs:<Container-ID>/<Object-ID/<Command>/<Params>".
 // If Command is not provided, full object is requested.
-func Get(ctx context.Context, priv *keys.PrivateKey, u *url.URL, addr string, resReader ResultReader) ([]byte, error) {
+func Get(ctx context.Context, priv *keys.PrivateKey, u *url.URL, addr string, resReader ResultReaderCloser) ([]byte, error) {
```

Comment:
I think you can return `io.ReadCloser()` here and drop `ResultReader` completely.